packages/relay-test-utils/package.json

```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "@babel/runtime": "^7.0.0",
     "fbjs": "^1.0.0",
+    "graphql-compose": "^5.8.0",
```

Comment:
Same question, we should use the built-in mechanism to extend schemas.

Reply:
It's my "lodash" for GraphQL type system 😈

If not today then it will be used tomorrow. Just give me some time to make this lib popular and right solution for such kind of problems.

Reply:
Which mechanism? extendSchema only supports extending through AST.